src/config/defaults.ts

```diff
@@ -18,7 +18,12 @@ export const defaults: Config = {
     disable: false,
     indexHTML: path.resolve(__dirname, '../admin/index.html'),
     avatar: 'default',
-    components: {},
+    components: {
+      logout: {
+        Route: '/logout',
+        InactivityRoute: '/logout-inactivity', 
```

Comment:
Hey @finkinfridom — everything is looking great, but I don't think the `Route` and `InactivityRoute` properties belong in the `components` object, because they are not components. I think that these should be added to the upper-level `admin` property, and be named something like `logoutRoute` and `inactivityRoute`. Can you move these? Then we will be done!

Reply:
hey @jmikrut I updated the code as per your request. I also updated the docs accordingly